client/src/components/HeroContact.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// import bgimage from "";
+// style={{ backgroundImage: `url(${bgimage})`, backgroundSize: 'cover' }}
+
+const Hero = (props) => {
+  return (
+    <div className="heroSection4">
```

Comment:
hero-contact or similar

Reply:
I can't add a comment for HeroSetup.js but there too like the other Heros

Reply:
Done